torch/utils/checkpoint.py

```diff
@@ -347,6 +347,7 @@ def checkpoint(
     context_fn: Callable[[], Tuple[ContextManager, ContextManager]] = noop_context_fn,
     determinism_check: str = _DEFAULT_DETERMINISM_MODE,
     debug: bool = False,
+    early_stop: bool = True,
```

Comment:
Can we make these kwargs only for the doc? Because passing them positionally won't work.

Reply:
TIL